test/e2e/providers/azure/azure.go

```diff
@@ -18,7 +18,7 @@ const (
 	defaultCredentialsSecretName = "azure-cloud-credentials"
 	defaultImageOffer            = "WindowsServer"
 	defaultImagePublisher        = "MicrosoftWindowsServer"
-	defaultImageSKU              = "datacenter-core-20h2-with-containers-smalldisk"
+	defaultImageSKU              = "2022-datacenter-smalldisk"
```

Comment:
See the container image for Azure:
https://github.com/openshift/windows-machine-config-operator/blob/3a633985ff99b0ea9565e5262c482480c9575f52/test/e2e/network_test.go#L455